web/src/host/pages/GameInProgress.jsx

```diff
@@ -13,32 +13,33 @@ export default function GameInProgress({
   questions,
   currentState,
   currentQuestionId,
-  handleChangeGameStatus,
   phaseOneTime,
   phaseTwoTime,
   handleUpdateGameSessionStateFooter
 }) {
 const classes = useStyles();
-const questionDetails = questions.items
- console.log(questionDetails);
 
   const stateArray = Object.values(GameSessionState); //adds all states from enum into array 
   let nextState;
  
   const numAnswersFunc = teams => { //finds all answers using isChosen, for use in footer progress bar
     let count = 0;
-    teams && teams.items.map(team => 
+    teams && teams.map(team => 
        team.teamMembers && team.teamMembers.items.map(teamMember => 
         teamMember.answers && teamMember.answers.items.map(answer => answer.isChosen && count++
     )))
 
     return count;
   };
 
+  if(currentQuestionId == null) {
```

Comment:
Haven't gone through the rest of code, so sorry if it's an obvious one, but why can't we have this to be `null`? Also for adding a default value we should use `questions[0].id` or something